tests/parser/globals/test_setters.py

```diff
@@ -125,11 +132,19 @@ def gop() -> int128:
         Zed({foo: [1, 2, 3], bar: [Bar({a: 4, b: 5}), Bar({a: 2, b: 3})]}),
         Zed({foo: [6, 7, 8], bar: [Bar({a: 9, b: 1}), Bar({a: 7, b: 8})]})
     ]
-    return zed[0].foo[0] + zed[0].foo[1] * 10 + zed[0].foo[2] * 100 + \
-        zed[0].bar[0].a * 1000 + zed[0].bar[0].b * 10000 + zed[0].bar[1].a * 100000 + zed[0].bar[1].b * 1000000 + \
-        zed[1].foo[0] * 10000000 + zed[1].foo[1] * 100000000 + zed[1].foo[2] * 1000000000 + \
-        zed[1].bar[0].a * 10000000000 + zed[1].bar[0].b * 100000000000 + \
-        zed[1].bar[1].a * 1000000000000 + zed[1].bar[1].b * 10000000000000
+    return zed[0].foo[0] + zed[0].foo[1] * 10 + \
+        zed[0].foo[2] * 100 + \
+        zed[0].bar[0].a * 1000 + \
+        zed[0].bar[0].b * 10000 + \
+        zed[0].bar[1].a * 100000 + \
+        zed[0].bar[1].b * 1000000 + \
+        zed[1].foo[0] * 10000000 + \
+        zed[1].foo[1] * 100000000 + \
+        zed[1].foo[2] * 1000000000 + \
+        zed[1].bar[0].a * 10000000000 + \
+        zed[1].bar[0].b * 100000000000 + \
+        zed[1].bar[1].a * 1000000000000 + \
+        zed[1].bar[1].b * 10000000000000
```

Comment:
@davesque you do see the irony here, one naturally uses the flake8 unpopular method of having operators at the end of the line (because it's vyper code).